src/main/docs/guide/opentelemetry.adoc

```diff
@@ -0,0 +1,57 @@
+The Micronaut Open Telemetry module uses https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md[Open Telemetry Autoconfigure SDK] to configure Open Telemetry for tracing. For some functionalities you have to add additional dependencies. The Default values that are defined inside the Micronaut which values might be different from the default ones inside https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md[Open Telemetry Autoconfigure SDK] module are:
+
+- otel.traces.exporter = none
+- otel.metrics.exporter = none
+- otel.logs.exporte = none
+- otel.service.name = value of the application.name
+
+== OpenTelemetry annotations
+The pkg:tracing.opentelemetry-annotation[] package contains transformers and mappers that enables usage of Open Telemetry annotations.
+
+To enable Open telemetry annotations you have to add next annotation processor in your dependency block:
+dependency:micronaut-management[scope="annotationProcessor", version="{version}", groupId="io.micronaut.tracing"]
```

Comment:
Is this dependency correct? Maybe you meant `micronaut-tracing-opentelemetry-annotation`

Reply:
nice catch 😄